onnx/defs/data_type_utils.cc

```diff
@@ -96,15 +97,19 @@ DataType DataTypeUtils::ToType(const TypeProto& type_proto) {
   std::lock_guard<std::mutex> lock(GetTypeStrLock());
   if (GetTypeStrToProtoMap().find(typeStr) == GetTypeStrToProtoMap().end()) {
     TypeProto type;
-    FromString(typeStr, type);
+    Common::Status st = FromString(typeStr, type);
+    if (!st.IsOK())
+      return nullptr;
     GetTypeStrToProtoMap()[typeStr] = type;
   }
   return &(GetTypeStrToProtoMap().find(typeStr)->first);
 }
 
 DataType DataTypeUtils::ToType(const std::string& type_str) {
   TypeProto type;
-  FromString(type_str, type);
+  Common::Status st = FromString(type_str, type);
+  if (!st.IsOK())
```

Comment:
Can we throw and print out the error message from `st` here?

Reply:
I prefer to change it to return Common::Status instead, to make it consist with the other function.  It look bad when mixing the use of Common::Status and exception.  
What do you think?

Reply:
I always prefer to propagate error to the surface. Let's return `Common::Status`. :)